### PR TITLE
Features/bgvkxv route mode codes

### PR DESCRIPTION
**Pull request by sarahjanenewnham-zz**

### CODE REVIEW

File: GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs [hunk 1]
```diff
@@ -193,7 +193,7 @@ public IEnumerable<Route> Get()
                     ShortName = x.IsDBNull(2) ? null : x.GetString(2),
                     LongName = x.IsDBNull(3) ? null : x.GetString(3),
                     Description = x.IsDBNull(4) ? null : x.GetString(4),
-                    Type = (RouteTypeExtended)x.GetInt64(5),
+                    Type = (RouteType)x.GetInt64(5),
```
Comment (peetjvv) [inline GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs:231]: I think you have to change this in the in-memory and postres database collections as well.

Comment (sarahjanenewnham-zz) [reply GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs:231]: Those seem to have been deleted from the `develop` branch. They're in master though?

Comment (timminata) [reply GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs:231]: MM, I'm on master on my local. So maybe master is more recent than develop?

Comment (peetjvv) [reply GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs:231]: master is the default branch in this repo

Comment (peetjvv) [reply GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs:231]: or at least we're using master and not develop

Comment (peetjvv) [reply GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs:231]: I'm working on fixing this

Comment (peetjvv) [reply GTFS.DB.SQLite/Collections/SQLiteRouteCollection.cs:231]: This should all be fixed now.
I merged in master, fixed the postgres db collections and changed the base branch for this PR to master.

**Review by peetjvv — APPROVED** [review]
tested locally. looks good

**Comment by timminata** [comment]
Looks good!

**Pull request merged**